src/main/java/Bob.java

```diff
@@ -0,0 +1,119 @@
+                displayHorizontalLine();
+                break;
+            case "event":
+                content = line.split(" ", 2)[1];
+
+                String description = content.split( " /from ")[0];
+                String start = content.split( " /from ")[1].split(" /to ")[0];
+                String by = content.split(" /to ")[1];
+
+                Event e = new Event(description, start, by);
+                list.add(e);
+
+                displayHorizontalLine();
+                System.out.println("Got it. I've added this task: ");
+                System.out.println(e);
+                System.out.println("Now you have " + list.size() + " tasks in the list.");
+                displayHorizontalLine();
+                break;
+            case "mark":
+                content = line.split(" ", 2)[1];
+                list.get(Integer.parseInt(content) - 1).setDone(true);
+                displayHorizontalLine();
+                System.out.println("Nice! I've marked this task as done: ");
+                System.out.println(list.get(Integer.parseInt(content) - 1).getListItem());
+                displayHorizontalLine();
+                break;
+            case "unmark":
+                content = line.split(" ", 2)[1];
+                list.get(Integer.parseInt(content) - 1).setDone(false);
+                displayHorizontalLine();
+                System.out.println("OK, I've marked this task as not done yet:");
+                System.out.println(list.get(Integer.parseInt(content) - 1).getListItem());
+                displayHorizontalLine();
+                break;
+            case "list":
+                displayList(list);
+                break;
+            case "bye":
+                displayExitMessage();
+                break loop;
+            default:
+                break;
+            }
+        }
+    }
+
+    private static void displayList(List<Task> list) {
+        displayHorizontalLine();
+        System.out.println("Here are the tasks in your list: ");
+        for (int i = 0; i < list.size(); i++) {
+            System.out.println((i+1) + "." + list.get(i).toString());
+        }
+
+        displayHorizontalLine();
+    }
+
+    private static void displayWelcomeMessage() {
+        String logo = " ____       _        \n"
+                    + "|  _ \\_____|_|__ _\n"
+                    + "| |_| | /\\ | |    \\ \n"
+                    + "| |_| | \\/ | |  O /\n"
+                    + "|____/ \\__,|_|\\__/\n";
+        System.out.println("Hello from\n" + logo);
+
+        displayHorizontalLine();
+        System.out.println("Hello! I'm Bob");
+        System.out.println("What can I do for you?");
+        displayHorizontalLine();
+    }
+
+    public static void displayExitMessage() {
+        displayHorizontalLine();
+        System.out.println("Bye. Hope to see you again soon!");
+        displayHorizontalLine();
+    }
+
+    private static void displayHorizontalLine() {
```

Comment:
I liked that you defined a method too print the horizontal line. It helps in enforcing the DRY principle.